amf_test.go

```diff
@@ -103,8 +102,7 @@ func TestUpdateConfig(t *testing.T) {
 	if err != nil {
 		panic(err)
 	}
-	var Rsp chan *protos.NetworkSliceResponse
-	Rsp = make(chan *protos.NetworkSliceResponse)
+	var Rsp chan *protos.NetworkSliceResponse = make(chan *protos.NetworkSliceResponse)
```

Comment:
What about the following for simplicity?
```suggestion
	Rsp := make(chan *protos.NetworkSliceResponse)
```